### PR TITLE
Some Basic Xamarin UI

### PR DESCRIPTION
**Pull request by Raeppa**
Written some basic UI with Raimond.
Includes:
Login Page
Registration Page
Bar/Drink Pages
Edit Page
About Page

### CODE REVIEW

File: WebService/ComparisonEngineUI/ComparisonEngineUI/Views/DrinksPage.xaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="ComparisonEngineUI.Views.DrinksPage">
+    <ContentPage.Content>
+        <StackLayout>
+            <Label TextColor="Black" Text="Drink List:"
+                VerticalOptions="Center" 
+                HorizontalOptions="Center" />
+            <Button Text="Edit" Command="{Binding EditDrinkCommand}"/>
+        </StackLayout>
```
Comment (pskramtai) [inline WebService/ComparisonEngineUI/ComparisonEngineUI/Views/DrinksPage.xaml:11]: We probably need some kind of scroll view to be able to fill this page with clickable buttons.

File: WebService/ComparisonEngineUI/ComparisonEngineUI/Views/BarsPage.xaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="ComparisonEngineUI.Views.BarsPage">
+    <ContentPage.Content>
+        <StackLayout>
+            <Label TextColor="Black" Text="Bar List:"
+                VerticalOptions="Center" 
+                HorizontalOptions="Center" />
+            <Button Text="Edit" Command="{Binding EditBarCommand}"/>
+        </StackLayout>
```
Comment (pskramtai) [inline WebService/ComparisonEngineUI/ComparisonEngineUI/Views/BarsPage.xaml:11]: Same thing as in DrinksPage.

**Comment by pskramtai** [comment]
Will there be any button binding in ComparisonEngineUI.Android activity files or will the entire logic be written in ComparisonEngineUI project? We could use delegates for button clicks, if we write some logic in activity files.

**Comment by Raeppa** [comment]
The whole logic will be in ComparisonEngineUI, ComparisonEngine.Android will be only for Android specific things

**Pull request merged**